boards/lilygo/ttgo_t8s3/ttgo_t8s3_appcpu.dts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32s3/esp32s3_appcpu.dtsi>
```

Comment:
can you please replace `esp32s3_appcpu.dtsi` with `#include <espressif/esp32s3/esp32s3_mini_n8.dtsi>`? The reason is that it has the flash node with a size attribute which is needed for target configuration.

Reply:
@marekmatej 
ok, done.
I peeked at m5stack/m5stack_cores3/m5stack_cores3_appcpu.dts
That included esp32s3_appcpu.dtsi